packages/addon-dev/src/rollup.ts

```diff
@@ -57,8 +57,8 @@ export class Addon {
     return hbs();
   }
 
-  gjs() {
-    return gjs();
+  gjs(options: { inline_source_map: true }) {
```

Comment:
@vstefanovic97 shouldn’t we mark this as optional?
```suggestion
  gjs(options?: { inline_source_map: true }) {
```

Reply:
🤦‍♂️ , I'll fix it right away